about.html

```diff
@@ -1 +1,135 @@
-<!-- Your markup here! -->
\ No newline at end of file
+	<!DOCTYPE html>
+	<html>
+
+		<head>
+		    <!-- Meta Stuff -->
+		  <meta charset="utf-8">
+
+		  <!-- Bootstap CSS -->
+		  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+
+		  <!-- Our CSS -->
+		  <link href="styles.css" rel="stylesheet">
+
+		</head>
+
```

Comment:
These tabs are out of control! I can show you how to configure SublimeText to make your indented code more readable.